charon/pkgs/npm.py

```diff
@@ -177,6 +196,10 @@ def handle_npm_del(
     )
     logger.info("Files deletion done\n")
 
+    logger.info("Start deleting manifest from s3")
+    client.delete_manifest(product, target, manifest_bucket_name)
+    logger.info("Manifest deletion is done\n")
```

Comment:
I think this step can be skipped with a log if manifest_bucket_name is not specified.